backend/module/eCampCore/src/EntityService/PeriodService.php

```diff
@@ -79,26 +82,26 @@ protected function createEntityPost(BaseEntity $entity, $data): Period {
      */
     protected function patchEntity(BaseEntity $entity, $data): Period {
         /** @var Period $period */
-        $period = parent::patchEntity($entity, $data);
-        $this->updatePeriodDays($period);
+        $period = $entity;
+        $oldStart = $period->getStart();
 
-        // $moveActivities = isset($data->move_activities) ? $data->move_activities : null;
-        // $this->updateScheduleEntries($period, $moveActivities);
+        // Validate start/end date
+        $this->validateInput($period, $data);
 
-        return $period;
-    }
-
-    /**
-     * @throws ORMException
-     * @throws NoAccessException
-     */
-    protected function updateEntity($id, $data): Period {
         /** @var Period $period */
-        $period = parent::updateEntity($id, $data);
+        $period = parent::patchEntity($period, $data);
         $this->updatePeriodDays($period);
 
-        // $moveActivities = isset($data->move_activities) ? $data->move_activities : null;
-        // $this->updateScheduleEntries($period, $moveActivities);
+        /** @var bool $moveScheduleEntries */
+        $moveScheduleEntries = isset($data->moveScheduleEntries) ? $data->moveScheduleEntries : false;
```

Comment:
Vielleicht das hier in eine Methode refactoren die an beiden Orten verwendet wird? Falls wir mal in ändern, welche Werte `true` und `false` repräsentieren kann das sonst leicht vergessen gehen, dass man es an beiden Orten anpassen muss.

Reply:
ist nicht mehr doppelt vorhanden, da validation umgebaut.